build-frontend/action.yml

```diff
@@ -2,47 +2,47 @@ name: Build Frontend
 description: Builds Storefront app docker image
 inputs:
   frontend:
-    description: 'Switch between Next.js and Nuxt'
+    description: "Switch between Next.js and Nuxt"
```

Comment:
please can we not change single/double quotes ? it is unecessary change, please adjust IDE